.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,13 @@
-https://github.com/nimblehq/flutter_templates/issues/??
+- Close #
 
 ## What happened 👀
 
-Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
+Provide a description of the **changes** this pull request brings to the codebase. Additionally, when the pull request is still being worked on, a checklist of the planned changes is welcome to track progress.
 
 ## Insight 📝
 
-Describe in details how to test the changes, which solution you tried but did not go with, referenced documentation is
-welcome as well.
+Describe in detail why this solution is the most appropriate, which solution you tried but did not go with, and how to test the changes. References to relevant documentation are welcome as well.
 
 ## Proof Of Work 📹
 
-Show us the implementation: screenshots, gif, etc.
-
-<img src="URL_GOES_HERE" width=200 />
```

Comment:
[Optional] I still think we should let the path `<img src="URL_GOES_HERE" width=200 />` in the template so we can use it quickly when creating the PR